int32 marker_id
geometry_msgs/Pose pose